core/src/main/java/com/microsoft/applicationinsights/internal/config/TelemetryConfigurationFactory.java

```diff
@@ -59,6 +61,8 @@
 
     private String performanceCountersSection = DEFAULT_PERFORMANCE_MODULES_PACKAGE;
 
+    final static String ENV_VARIABLE_I_KEY = "APPLICATION_INSIGHTS_IKEY";
```

Comment:
Regarding the constant name then this is no longer only the name of the environment variable but also for the system variable.
Therefore consider renaming it to something like `EXTERNAL_PROPERTY_IKEY` or `EXTERNAL_PROPERTY_IKEY_NAME`